lms/djangoapps/instructor/tests/test_api.py

```diff
@@ -2236,6 +2237,249 @@ def test_rescore_problem_all(self, act):
         self.assertEqual(response.status_code, 200)
         self.assertTrue(act.called)
 
+    @patch.dict(settings.FEATURES, {'ENTRANCE_EXAMS': True})
+    def test_course_has_entrance_exam_in_student_attempts_reset(self):
+        """ Test course has entrance exam id set while resetting attempts"""
+        url = reverse('reset_student_attempts_for_entrance_exam',
```

Comment:
Nit: Yea, in order to handle long lines, the standard is to have an open '(' on one line, then all of the parameters as individual lines with one indent in from the '('. Then the closing ')' would not be indented. If you look in the edx-notifications repo, I'm pretty stringently following that pattern.